native/src/rendering.cpp

```diff
@@ -410,6 +411,7 @@ void renderText(MapDataObject* obj, RenderingRuleSearchRequest* req, RenderingCo
 							fillTextProperties(rc, dupInfo, req, x, y);
 							dupInfo->secondOrder = ((obj->id % 10000) << 8) + k;
 							rc->textToDraw.push_back(dupInfo);
+							renderableObjects.emplace(dupInfo->object.id, std::move(rObj));
```

Comment:
All renderableObjects.emplace should have a flag to add or not to add ? (to not slow down in case it's not needed)